sw/device/lib/runtime/ibex.h

```diff
@@ -27,15 +27,28 @@ extern const size_t kIbexClockFreqHz;
  *
  * @param cycles the number of cycles to burn.
  */
-inline void ibex_busy_loop(size_t cycles) {
-  size_t out;  // Used to create an inout parameter below.
+inline void ibex_busy_loop(uint64_t cycles) {
```

Comment:
I can't comment further up in the docstring, but typical behavior in `usleep()` and friends is to spin for *at least* the amount of cycles.

Reply:
Using the cycle counter resolves this issue.